hs_core/xmlparser.py

```diff
@@ -4,6 +4,7 @@
 import os.path
 from io import StringIO
 from xml.sax import xmlreader, saxutils
+# TODO: should we use defusedxml.sax?
```

Comment:
I'm not sure if this is a security concern.

Reply:
It shouldn't hold up this PR.  Create an issue to investigate and update there if needed.

Reply:
will do